.cloudbees/testing/action.yml

```diff
@@ -54,6 +59,14 @@ runs:
       run: |
         set -uo pipefail
 
+        if echo "${{ inputs.chart-location }}" | grep -q '^oci://'; then
```

Comment:
To be safe, it would be good to also use this input within the script via an env var.